eng/Version.Details.xml

```diff
@@ -29,21 +29,21 @@
       <Uri>https://github.com/microsoft/usvc-apiserver</Uri>
       <Sha>9582c787e7963a2acf11111e5bc5a4938e7fe3ab</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Http.Resilience" Version="9.0.0-preview.9.24524.6">
-      <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>7fe79a39d7e28f46eaa51d82a3d433bff73bd3cd</Sha>
+    <Dependency Name="Microsoft.Extensions.Http.Resilience" Version="9.0.0">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-extensions</Uri>
```

Comment:
Should this really be pointing to an internal repo? I see we do it for dotnet/runtime as well (existing).

Reply:
This is what darc adds automatically. For darc, it knows how to handle the mapping between internal and external repos, so that won't impact at all the behavior of dependency flow, so when you flow internal builds, you get internal repos, when you flow external builds, you get external repos.

Reply:
A while back I would always translate back to the public repo on the merge back to public, but after chatting with @mmitche he suggested we just leave it as is.